api/anubis/views/public/auth.py

```diff
@@ -89,6 +96,52 @@ def public_oauth():
     return r
 
 
+@github_oauth_.route("/link")
+@require_user()
+def public_github_link():
+    return github_provider.authorize(
+        callback="http://localhost:3000/api/public/github/oauth"
```

Comment:
This needs to change to what it will be on prod. I think we could maybe read the HOST http header to determine if `localhost` or `anubis.orisis.services` should be used.

Reply:
Yeah definitely. I just put there for testing purpose.